tools/installPSResources.ps1

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+$ErrorActionPreference = 'Stop'
+
+Set-PSRepository -Name PSGallery -InstallationPolicy Trusted | Out-Null
+if ($PSVersionTable.PSVersion.Major -lt 6) {
+    throw "The build script requires PowerShell 7!"
+}
+
+# TODO: Switch to Install-PSResource when CI uses PowerShell 7.4
+Install-Module -Name PowerShellGet -AllowPrerelease -Scope CurrentUser -Force
+Install-Module -Name Pester -RequiredVersion 4.10.1 -Scope CurrentUser -Force
```

Comment:
This was the annoying bit and why I couldn't get these to run locally. Pester 5 (the latest, and what installs by default) is not backwards compatible, so this system for now is stuck with 4.x. Thanks Patrick 😆

Reply:
Should use PSResourceGet instead

Reply:
I think that will need to be done separately because this project's build tools specifically require this:

https://github.com/PowerShell/SecretManagement/blob/47bc3b71a810e0810aa2fc2bff645ffb7e5f90b3/buildtools.psm1#L5

But noted for a separate PR.